generator/osm2meta.cpp

```diff
@@ -384,3 +549,22 @@ string MetadataTagProcessorImpl::ValidateAndFormat_duration(string const & v) co
   return format(hours);
 }
 
+std::string MetadataTagProcessor::FormatFacebook(std::string &v) const
```

Comment:
```suggestion
string MetadataTagProcessor::FormatFacebook(std::string const & v) const
```

Note: we don't use std:: in cpp files if there is already `using namespace std;` above.

Reply:
The same below.